test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -2627,27 +2671,34 @@ def test_cgroup_execjob_end_should_delete_cgroup(self):
         self.server.expect(JOB, 'queue', id=jid, op=UNSET,
                            interval=1, offset=1)
         # verify that cgroup files for this job are gone even if
-        # epilogue and periodic events are not disabled
+        # epilogue and periodic events are disabled
         for subsys, path in self.paths.items():
             # only check under subsystems that are enabled
             enabled_subsys = ['cpuacct', 'cpuset', 'memory', 'memsw']
             if (any([x in subsys for x in enabled_subsys])):
                 continue
             if path:
-                # Apparently cgroup_prefix for this test is pbs
-                filename = os.path.join(path, 'pbs', str(jid))
-                self.logger.info('Checking that file %s should not exist'
-                                 % filename)
-                self.assertFalse(os.path.isfile(filename))
-                filename = os.path.join(path, 'pbs.slice', 'pbs-%s.slice'
-                                        % systemd_escape(jid))
-                self.logger.info('Checking that file %s should not exist'
-                                 % filename)
-                self.assertFalse(os.path.isfile(filename))
-                filename = os.path.join(path, 'pbs.service', str(jid))
+                filename = os.path.join(path,
+                                        'pbs_jobs.service',
+                                        'jobid', str(jid))
                 self.logger.info('Checking that file %s should not exist'
                                  % filename)
                 self.assertFalse(os.path.isfile(filename))
+            #   filename = os.path.join(path, 'pbs.service',
+            #                           'jobid', str(jid))
+            #    self.logger.info('Checking that file %s should not exist'
+            #                     % filename)
+            #    self.assertFalse(os.path.isfile(filename))
+            #    filename = os.path.join(path, 'pbs.slice', 'pbs-%s.slice'
+            #                            % systemd_escape(jid))
+            #    self.logger.info('Checking that file %s should not exist'
+            #                     % filename)
+            #    self.assertFalse(os.path.isfile(filename))
+            #    filename = os.path.join(path, 'pbs.slice', 'pbs-%s.slice'
+            #                            % systemd_escape(jid))
+            #    self.logger.info('Checking that file %s should not exist'
+            #                     % filename)
+            #    self.assertFalse(os.path.isfile(filename))
```

Comment:
Please remove commented code.

Reply:
That was left there for a reason, to make it easier to reuse the test for an older hook.

Since get_cgroup_job_dir already lists the other possible paths, replaced with a comment containing a reference to that function.